src/sentry/tasks/groupowner.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import absolute_import, print_function
+
+import logging
+
+from datetime import timedelta
+
+from django.core.cache import cache
+from django.utils import timezone
+
+from sentry.eventstore.models import Event
+from sentry.eventstore.processing import event_processing_store
+from sentry.models import Project, Release
+from sentry.models.groupowner import GroupOwner, GroupOwnerType
+from sentry.tasks.base import instrumented_task
+from sentry.utils.committers import get_serialized_event_file_committers
+
+PREFERRED_GROUP_OWNERS = 3
+PREFERRED_GROUP_OWNER_AGE = timedelta(days=1)
+
+logger = logging.getLogger("tasks.groupowner")
+
+
+@instrumented_task(
+    name="sentry.tasks.process_suspect_commits", default_retry_delay=5, max_retries=5
+)
+def process_suspect_commits(group_id, cache_key, **kwargs):
+    if not group_id:
+        return
+
+    can_process = True
```

Comment:
Since we set this to `True` by default, it looks like we will constantly recalculate suspect commits for groups that don't have any.